### PR TITLE
[1] Filter list

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into Cypress require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/views/records/_records_list.html.erb [hunk 1]
```diff
@@ -27,15 +31,15 @@
...
       <% if measure && !hide_patient_calculation?%>
+        <th scope="col">Population Group</th>
         <% population_keys.each do |population| %>
           <th scope="col" class="text-center"><%= population_label(bundle, population) %></th>
         <% end %>
       <% end %>
     </tr>
   </thead>
-  <% if measure %>
+  <% if measure&.name == 'what' %>
```
Comment (lmd59) [inline app/views/records/_records_list.html.erb]: what's happening here?

**Review by lmd59 — APPROVED** [review]
